zeppelin-web/package.json

```diff
@@ -40,50 +38,53 @@
     "angular-sanitize": "1.5.7",
     "angular-touch": "1.5.7",
     "angular-ui-ace": "0.2.3",
-    "angular-websocket": "1.0.14",
+    "angular-ui-bootstrap": "~2.5.0",
+    "angular-ui-grid": "4.4.6",
+    "angular-websocket": "1.1.0",
```

Comment:
Is this an intentional minor update?

Reply:
No, it wasn't. 

I think there was a mistake during rebase. 

Fixed! 
https://github.com/Henry-Hong/zeppelin/blob/e2b12d1a5a53f17c65a87ac9997b294aeff946b0/zeppelin-web/package.json#L43